docs/application/tizen-studio/release-notes/3-3-release-notes.md

```diff
@@ -0,0 +1,67 @@
+# Tizen Studio 3.3 Release Notes
+
+-   Release Date: July 26, 2019
```

Comment:
The expected target date is July 24.
So would you please change to "July 24, 2019"?

Reply:
Updated the date.